scripts/build_virtualenv.sh

```diff
@@ -149,7 +149,11 @@ if [ ! -e $VENV/inst/.completed.$SCRIPT_VERSION ]; then
     if [ ! -e $VENV/inst/bin/python ]; then
         pushd cpython-2011
         hg checkout dtrace-issue13405_2.7
-        LDFLAGS="-Wl,-rpath=$VENV/inst/lib" ./configure $EXTRA_CONFIG_OPTS --prefix=$VENV/inst --enable-shared
+        if ( python -c 'import sys; print(sys.maxunicode)' | grep -q '65535' ); then
+            LDFLAGS="-Wl,-rpath=$VENV/inst/lib" ./configure $EXTRA_CONFIG_OPTS --prefix=$VENV/inst --enable-shared
+        else
+            LDFLAGS="-Wl,-rpath=$VENV/inst/lib" ./configure $EXTRA_CONFIG_OPTS --prefix=$VENV/inst --enable-shared --enable-unicode=ucs4
```

Comment:
I don't know why I wrote LDFLAGS, sorry. I meant the (whole) command line, something like:
``` bash
if CONDITION then EXTRA_PYTHON_CONFIG_ARGS="asdfasdf"
fi
LDFLAGS=foo bar ./configure --whatever $EXTRA_PYTHON_CONFIG_ARGS
```